acceptance/bundle/templates/telemetry/custom-template/script

```diff
@@ -6,6 +6,9 @@ mv output/my_custom_template/databricks.yml out.databricks.yml
 rm -r output
 
 
+# Disable pipefail. head will skip reading all input once it encounters a newline. Not disabling pipefail will trigger
+# a SIGPIPE in linux based systems if jq outputs more than one line.
+set +o pipefail
 cmd_exec_id=$(cat out.requests.txt | jq '.headers."User-Agent".[0]'| head -n 1 | grep -o 'cmd-exec-id/[^ ]*' | cut -d '/' -f2)
```

Comment:
why does it mean to grep after 'head -n 1'? There is only one line, what are you filtering?

This pipeline is quite complex, perhaps it's better expressed in a small python script?

Reply:
> This pipeline is quite complex, perhaps it's better expressed in a small python script?

Maybe. I do like how this approach forces you to get familiar with common unix tools and is quite powerful once you get used to it. A python script would be much more verbose and introduce a level of indirection. So I'd prefer the existing approach.

Reply:
but without pipefail you lose error handling completely from any inner command. Also, if input does not match, you won't get any good diagnostics there.